packages/node-resolve/src/index.js

```diff
@@ -261,9 +260,13 @@ export function nodeResolve(opts = {}) {
         importer = undefined;
       }
 
-      const resolved = await doResolveId(this, importee, importer, opts);
+      const resolved = await doResolveId(this, importee, importer, resolveOptions.custom);
```

Comment:
This commit broke plugin-node-resolve, see: https://github.com/rollup/plugins/issues/1023

Reply:
opened https://github.com/rollup/plugins/pull/1029